src/test_DisplayMenu.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+import IndexGame
+
+def test_displayMainMenu(capfd): #Test display menu with return value of the menu display
+    IndexGame.displayMainMenu()
+    out, _ = capfd.readouterr()
+    assert\
+    ("Welcome, mayor of Simp City!\n" +
+    "------------------------------\n" +
+    "1. Start new game\n" +
+    "2. Load saved game\n" +
+    "3. Show high scores\n" +
+    "0. Exit")\
+    in out
+
+def test_runMainMenu(capfd, monkeypatch): #insert monkeypatch with expected outcome
+    monkeypatch.setattr('builtins.input', lambda _: "0")
+    expected = "Welcome, mayor of Simp City!\n\
+------------------------------\n\
+1. Start new game\n\
+2. Load saved game\n\
+3. Show high scores\n\
+0. Exit\n\
+Thanks for playing!\n"
+    IndexGame.runMainMenu() #main menu runs here
+    out, _ = capfd.readouterr()
+    assert out == expected
+
+def test_exit(capfd): #test exit game input
+    result = IndexGame.exitGame()
+    assert result == "Thanks for playing!"
+    
+    
+@pytest.mark.parametrize("userinput, expectedprint",
```

Comment:
As mentioned, we are testing for the output of the function, not so much of the display yet. Will work into testing the displaying of the function when the foundations has been laid